openshift-templates/hubot/template.json

```diff
@@ -0,0 +1,270 @@
+                                        "value": "${HUBOT_SLACK_TOKEN}"
+                                    },
+                                    {
+                                        "name": "HUBOT_JENKINS_URL",
+                                        "value": "${HUBOT_JENKINS_URL}"
+                                    },
+                                    {
+                                        "name": "NODE_TLS_REJECT_UNAUTHORIZED",
+                                        "value": "${NODE_TLS_REJECT_UNAUTHORIZED}"
+                                    },
+                                    {
+                                        "name": "HUBOT_JENKINS_AUTH",
+                                        "value": "${HUBOT_JENKINS_AUTH}"
+                                    }
+                                ],
+                                "resources": {},
+                                "terminationMessagePath": "/dev/termination-log",
+                                "terminationMessagePolicy": "File",
+                                "imagePullPolicy": "Always"
+                            }
+                        ],
+                        "restartPolicy": "Always",
+                        "terminationGracePeriodSeconds": 30,
+                        "dnsPolicy": "ClusterFirst",
+                        "serviceAccountName": "${SERVICE_ACCOUNT}",
+                        "serviceAccount": "${SERVICE_ACCOUNT}",
+                        "securityContext": {},
+                        "schedulerName": "default-scheduler"
+                    }
+                }
+            }
+        },
+        {
+            "kind": "Service",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "hubot",
+                "labels": {
+                    "app": "hubot"
+                }
+            },
+            "spec": {
+                "ports": [
+                    {
+                        "name": "8080-tcp",
+                        "protocol": "TCP",
+                        "port": 8080,
+                        "targetPort": 8080
+                    }
+                ],
+                "selector": {
+                    "deploymentconfig": "hubot"
+                },
+                "type": "ClusterIP",
+                "sessionAffinity": "None"
+            }
+        },
+        {
+            "kind": "Route",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "hubot",
+                "labels": {
+                    "app": "hubot"
+                }
+            },
+            "spec": {
+                "to": {
+                    "kind": "Service",
+                    "name": "hubot",
+                    "weight": 100
+                },
+                "port": {
+                    "targetPort": "8080-tcp"
+                },
+                "wildcardPolicy": "None"
+            }
+        }
+    ],
+    "parameters": [
```

Comment:
can you double check these parameters? I think all of them are required and should be marked as such